config/default.go

```diff
@@ -61,6 +61,7 @@ func (c *SnapshotterConfig) FillUpWithDefaults() error {
 	if daemonConfig.NydusdConfigPath == "" {
 		daemonConfig.NydusdConfigPath = defaultNydusDaemonConfigPath
 	}
+	daemonConfig.RecoverPolicy = "restart"
```

Comment:
Maybe make `restart`, `failover` as constant?

Reply:
We already have the ReoverPolicy custom type definition which is an alias of int. So let's use the parse String() method here rather literal string